src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java

```diff
@@ -291,11 +292,13 @@ private boolean canGenerateRequestsInFuture( WorkflowDocument workflowDoc, Strin
 
 		org.kuali.rice.krad.workflow.service.WorkflowDocumentService workflowDocumentService = KRADServiceLocatorWeb.getService( "workflowDocumentService" );
 		String currentRouteNodeNames = workflowDocumentService.getCurrentRouteNodeNames( workflowDoc );
-
-		return ( hasAskedToNotReceiveFutureRequests( workflowDoc, principalId ) && canGenerateMultipleApprovalRequests( reportCriteriaBuilder.build(), principalId, currentRouteNodeNames ) );
+		boolean result = hasAskedToReceiveFutureRequests( workflowDoc, principalId ) && canGenerateMultipleApprovalRequests( reportCriteriaBuilder.build(), principalId, currentRouteNodeNames );
+		LOG.info("canGenerateRequestsInFuture currentRouteNodeNames="+currentRouteNodeNames+" principalId="+principalId+ " result="+result);
```

Comment:
Should this be LOG.debug instead? Any concerns about the number of log entries caused by the LOG.info?